src/sentry/static/sentry/app/utils/withOrganization.tsx

```diff
@@ -29,4 +29,11 @@ const withOrganization = <P extends InjectedOrganizationProps>(
     }
   };
 
+export function isLightweightOrganization(
+  organization: Organization | LightWeightOrganization
+) {
```

Comment:
Should this function use `is` and be a [type predicate](https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates)?

Reply:
Ooo this much better. Good catch.